spec/statement.dd

```diff
@@ -1163,14 +1163,31 @@ switch (name)
         less error prone. char, wchar and dchar strings are allowed.
         )
 
-        $(P $(D Implementation Note:) The compiler's code generator may
+        $(P If a $(GLINK CaseStatement), $(GLINK CaseRangeStatement), or $(GLINK DefaultStatement)
+        appears in the top level of the $(GLINK ScopeStatement), then a new scope for local
+        symbols is introduced immediately following it, extending to the first appearance of one of
+        $(GLINK CaseStatement), $(GLINK CaseRangeStatement), $(GLINK DefaultStatement), `}`,
+        the end of a $(GLINK MixinStatement), or the end of the file.
```

Comment:
The end of the file cannot be reached anyway as the switch statement is inside part of a function body (or similar).